ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -0,0 +1,91 @@
+username shell ssh-key secondary ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDHMTFuLHPz/prREZZIks0ca4btBIzEbvY6KRYGzhN7JCG5CTfre0Y9UCbNul7qNl7cxomQkh/0VjQNX6ecPd0HyOTKL2EK002ejNyvooUDarnglMWtjKIl40NgDR/GNSkvC3nEylvX1H7Rfmu38NCqiwIpWA8JFwgLCLvkWUoORxHhIIy8/vttLgMxr66HGlVAnRidf3VVCnlILm4gUpc3fR43EhvVoYByY3jEa/fypiS2nDP9K2fXtpXGrIHSbyMu4Mj3fnSdcqWysRF7Tqc6Kvet8ImS07fLcgpbdLp31ssF1rssbTnD1zWuAozvXpK1d+vFO4EfFr5yzkE2Q8lM0wPpdS4LBWQfJdWgi6t5XEXewWyTYfIDKCBOI2dECGtkDjme+PDNIL9IQiiYC2iXMmQrun9fsp8jicdw1svGef8Otdb4kmHXiQ3mAxTeHLgeYPfYyekKq/+dFMcAZT+sv0g24AHc4ulitfLRoGjxYHZLGg2KQpFfAn0aQKCd5vk= noname@hostmachine-asd-cl
+!
+aaa group server ldap LADP2
+   server 10.10.10.157 vrf mgt
+   server 10.10.10.249
+!
+aaa group server ldap LDAP1
+   server 192.168.10.157 vrf mgt
+   server 10.10.10.248
+!
+radius-server attribute 32 include-in-access-req hostname
+radius-server dynamic-authorization port 1700
+radius-server tls ssl-profile GLOBAL_RADIUS_SSL_PROFILE
+radius-server dynamic-authorization tls ssl-profile SSL_PROFILE
+radius-server host 10.10.10.157 vrf mgt key 7 071B245F5A
+radius-server host 10.10.10.249 key 7 071B245F5A
+radius-server host 10.10.10.158 key 7 071B245F5A
+radius-server host 10.10.11.157 vrf mgt timeout 1 retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.159 vrf mgt retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.160 vrf mgt timeout 1 key 7 071B245F5A
+radius-server host 10.10.11.248 vrf mgt key 7 071B245F5A
+radius-server host 10.10.11.249 timeout 1 retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.158 timeout 1 retransmit 1 key 7 071B245F5A
+radius-server host 10.10.11.156 tls port 1700 timeout 1 retransmit 1
+radius-server host 10.10.11.155 vrf mgt tls ssl-profile HOST_SSL_PROFILE port 2083 timeout 1 retransmit 1
+!
+aaa group server radius RADIUS1
+   server 192.168.10.157 vrf mgt
+   server 10.10.10.248
+!
+aaa group server radius RADIUS2
+   server 10.10.10.157 vrf mgt
+   server 10.10.10.249
+!
+tacacs-server timeout 10
+tacacs-server host 10.10.10.157 single-connection vrf mgt key 7 071B245F5A
+tacacs-server host 10.10.10.249 timeout 23 key 7 071B245F5A
+tacacs-server host 10.10.10.158 key 7 071B245F5A
+tacacs-server host 10.10.10.159 key 8a $kUVyoj7FVQ//yw9D2lbqjA==$kxxohBiofI46IX3pw18KYQ==$DOOM0l9uU4TrQt2kyA7XCKtjUA==
+!
+aaa group server tacacs+ TACACS
+   server 10.10.11.157 vrf mgt
+   server 10.10.11.249
+!
+aaa group server tacacs+ TACACS1
+   server 10.10.10.157 vrf mgt
+   server 10.10.10.249
+!
+aaa group server tacacs+ TACACS2
+   server 192.168.10.157 vrf mgt
+   server 10.10.10.248
+!
+aaa authentication login default group TACACS local
+aaa authentication login console local
+aaa authentication enable default group TACACS local
+aaa authentication dot1x default group TACACS
```

Comment:
@alexeygorbunov @gmuloc @ClausHolbechArista 
Need help to configure this command. It require to create the dot1x group first but am unable to get it in my lab.
s1-host1(config)#aaa authentication dot1x default group TACACS
% group TACACS does not support dot1x authentication

Reply:
I think TACACS+ is simply not supported by dot1x:
```
cdv116...19:43:51(config-s-s1)#aaa authentication dot1x default group ?
  WORD    Server-group name
  radius  Use list of all defined RADIUS hosts
```

https://www.arista.com/en/um-eos/eos-control-plane-security

```
Configuring 802.1X Port Security
Basic steps to implementing 802.1X Port-based Network Access Control and RADIUS accounting on the switch:

A RADIUS server is required on one or more of your network servers or management stations. 802.1X is not supported with the TACACS+ authentication protocol.
```

Reply:
Changed to Radius group now its working